nullaway/src/main/java/com/uber/nullaway/handlers/InferredJARModelsHandler.java

```diff
@@ -109,21 +101,23 @@ public InferredJARModelsHandler(Config config) {
     }
   }
 
-  /*
-   * Scan Java Classpath for JarInfer model jars and map their names to locations
+  /**
+   * Loads all stubx files discovered in the classpath. Stubx files are discovered via
```

Comment:
Before there was some kind of scheme to lazily load stubx files (I think).  But I couldn't understand it, and it doesn't really fit with the service loader approach.  So now all provided stubx files are loaded eagerly (which I think shouldn't cause any real perf issues).

Reply:
Yes, that's correct that astubx files right now are only used for Android SDK and the test project.